package.json

```diff
@@ -6,7 +6,7 @@
     "test": "tests"
   },
   "scripts": {
-    "test": "bender run",
+    "test": "bender run -l && bender run -b firefox",
```

Comment:
Phantom does some sketchy things around walking in selection ranges (the first heading always seems to go missing), this behaviour does not occur in chrome & firefox.